sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -34,6 +41,7 @@ export interface GetTokenOptions {
     };
     tracingOptions?: {
         spanOptions?: SpanOptions;
+        context?: Context;
```

Comment:
I really wish we didn't have to drag in so many types for this

Reply:
Yeah, this one's difficult. They have two separate parameters (and no structure that joins them together) so we can't even create an opaque "holder" for this.